src/App.tsx

```diff
@@ -59,14 +59,16 @@ export const App = () => {
     resetForm();
   };
 
+  const validTitle = newTitle.replace(/[^a-zA-Z ]/g, '');
```

Comment:
no problem but you can use test() for condition.

`/[^a-zA-Z ]/g.test(newTitle);`